Feature

tasnimulhasan07 opened the pull request:
create mobile skeleton by recreating repo using project one knowledge, and keeping the code well organized using flexbox. added container and items onto Html and used CSS to customize them and give them an appearance, also fixed multiple linter errors.

tasnimulhasan07 committed:
final pull merge fix with branch

tasnimulhasan07 committed:
add contact icon

tasnimulhasan07 committed:
add github icon with link

tasnimulhasan07 committed:
images for background

tasnimulhasan07 committed:
remaining icons with links for social media etc

tasnimulhasan07 committed:
add full header and body for main page and build mobile suitability

tasnimulhasan07 committed:
use flexbox to position all items and containers in page

tasnimulhasan07 committed:
cmder amendment from pull request linter error

tasnimulhasan07 committed:
fixed linter error

tasnimulhasan07 committed:
fixed / error linters

tasnimulhasan07 committed:
finally syntax error identified

tasnimulhasan07 committed:
fix css linters errors

tasnimulhasan07 committed:
Update style.css

tasnimulhasan07 committed:
fixed 2 errors css

mmsesay commented on index.html:
```diff
@@ -0,0 +1,75 @@
+    <header>
+    <nav>
+    <div class="navbar">
+            <h1 class="logo"> My Logo </h1>
```
- [ ] Good implementing the logo. I suggest you set it above the `nav` tag and replace the `h1` with an `a` tag. Normally, a site's logo will enable a user to navigate back to the homepage. It will be good if you could fix this.

See example
```
<header>
   <a> // you logo </a>
   <nav> // your hamburger icon </nav>
</header/>
```

mmsesay commented on index.html:
```diff
@@ -0,0 +1,75 @@
+    <div class="navbar">
+            <h1 class="logo"> My Logo </h1>
+
+            <ul>
+              <li class="nav-items"><a href="#">Portfolio</a></li>
+              <li class="nav-items"><a href="#">About</a></li>
+              <li class="nav-items"><a href="#">Contact</a></li>
+            </ul>
```
- [ ] [OPTIONAL] These navigations will be required later on in the desktop version. I suggest you keep them out for now.

mmsesay commented on index.html:
```diff
@@ -0,0 +1,75 @@
+            <br>
+            <br>
+            <p>
+              I am a software developer. I have done some interesting projects using
```
- [ ] This title `I am a software developer.` is a sub-title. In that case, I suggest you place it in it a separate tag like `h2` and above the `p` tag.

```
<h1> // main title </h1>
<h2> // sub-title </h2>
<p> // paragraph </p>
```

mmsesay commented on style.css:
```diff
@@ -0,0 +1,178 @@
+section h1 {
+  display: flex;
+  position: static;
+  font-family: sans-serif;
```
- [ ] Please use the correct font as described in your chosen template. I think the font is called font `Inter`. Please don't forget to use a generic font like `sans-serif` or `serif` alongside the external fonts. If you don't, you might get some styling linter errors relating to the generic font family.

mmsesay commented on style.css:
```diff
@@ -0,0 +1,178 @@
+
+@media (max-width: 800px) {
+  .checkbtn {
+    display: block;
+  }
+}
+
+.nav-menu {
+  position: fixed;
+  left: 100%;
+  top: 70px;
+  gap: 0;
+  flex-direction: row;
+  background-color: #6070ff;
+  width: 100%;
+  text-align: left;
+  transition: 0.3s;
+}
+
+.nav-menu.active {
+  left: 0;
+}
+
+.socials {
+  color: #7fbcff;
+  font-size: 16px;
+  padding-bottom: 20px;
+}
+
+.socials:hover {
+  background: #acb7c3;
+  transition: 0.4s;
+}
+
+.icon {
+  width: 20px;
+  margin: 2px;
+  display: flex;
+  flex-direction: row;
+}
```
- [ ] [OPTIONAL] Good job implementing a media query. However, this was not a requirement. I suggest you keep your scope around the tasks requirements.

mmsesay commented on index.html:
```diff
@@ -0,0 +1,75 @@
+          <section class="homepage">
+            <div class="intro">
+
+            <h1>I'm Tasnimul <br>Glad to meet you!</h1>
+            <br>
+            <br>
+            <p>
+              I am a software developer. I have done some interesting projects using
+              react js, node js and ruby on rails. I am learning new things and
+              implementing them instantly. I can help you build a project, feature
+              or website look through some of work and experience if you like what
+              you see and have a project you need coded, don't hesitate to contact
+              me.
+            </p>
+
+            <h2 class="socials"> Let's Connect</h2>
+                       <ul class="icon">
+                           <li><a href="#"><img src="image/angellist.svg" alt="" /></a></li>
+                           <li><a href="#"><img src="Image/github.svg" alt="" /></a></li>
+                           <li><a href="#"><img src="Image/medium.svg" alt="" /></a></li>
+                           <li><a href="#"><img src="Image/twitter.svg" alt="" /></a></li>
+                       </ul>
+                   </div>
+               </section>
+
+
+
+               <script src="./app.js"></script>
+
+
+
+
+
+
+
+
+
+
+
+</body>
```
- [ ] You did well working on this project. But so far, I will suggest you always try to stick with the Figma design and specifications. Currently, your design differs a bit from the Figma design.

Your Design | Figma Design |
:---------------:|:-------------------:|
![c1](https://user-images.githubusercontent.com/39523312/148653962-9f3aca53-0e5c-4bc8-a292-42670d64ece0.png) |![c2](https://user-images.githubusercontent.com/39523312/148653965-cb21186f-718f-4d95-b774-cb31e7b94ab9.png)

mmsesay reviewed (changes requested):
Hi @tasnimulhasan07 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tasnimulhasan07 committed:
hamburger js

added additional js code to hamburger funtion

tasnimulhasan07 committed:
export image

exported image from figma templates

tasnimulhasan07 committed:
mobile portfolio

code additional page with buttons headers paragraphs while following the figma templates

tasnimulhasan07 committed:
pages css

use flexbox and basic css to create basic properties for all containers and items for pages, including font properties, positions of each item and container using flex
